packages/smooth_app/lib/views/bottom_sheet_views/social_handle_view.dart

```diff
@@ -0,0 +1,58 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+import 'package:smooth_app/generic_lib/widgets/smooth_list_tile.dart';
+import 'package:smooth_app/helpers/launch_url_helper.dart';
+
+class SocialHandleView extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Material(
+      child: SizedBox(
+        height: MediaQuery.of(context).size.height * 0.8,
+        child: ListView(
+          shrinkWrap: true,
+          scrollDirection: Axis.vertical,
+          children: <Widget>[
+            Container(
```

Comment:
To enhance a little bit your code, could you make this Container a dedicated Widget?

Reply:
Yeah, Sure.